### PR TITLE
Updated python to 3.6

### PR DESCRIPTION
**Pull request by MacMcIrish**
Also refactored test for coverage ( link / unlink )

### CODE REVIEW

File: tests/core/test_ModelFilter.py [hunk 1]
```diff
@@ -132,10 +132,14 @@ def test_filter_list_is_and_query_for_relationship_to_many(self, Teacher):
         assert Teacher.filter({
             'students.id': [self.student1.id, self.student2.id]
         }).first() is not None
+        self.unlink(student=self.student2.id, teacher=self.teacher1.id)
         assert Teacher.filter({
-            'students.id': [self.student1.id, self.student3.id]
+            'students.id': [self.student1.id, self.student2.id]
```
Comment (simlu) [inline tests/core/test_ModelFilter.py]: why did you change this?

Comment (MacMcIrish) [reply tests/core/test_ModelFilter.py]: Just as a way to use the link / unlink and gain some more coverage
